src/Illuminate/Support/Str.php

```diff
@@ -2083,4 +2083,17 @@ public static function flushCache()
         static::$camelCache = [];
         static::$studlyCache = [];
     }
+
+    /**
+     * Split a string by a given separator.
+     *
+     * @param  string  $separator
+     * @param  string  $string
+     * @param  int  $limit
+     * @return array
```

Comment:
The `array` will either consist of strings or be empty, so we can narrow this further as well:
```suggestion
     * @return string[]
```

Reply:
Ah, damn, sorry, hadn't submitted it and didn't see that it was closed when I hit the 'submit' button an hour later 😅